packages/next/src/server/app-render/types.ts

```diff
@@ -304,8 +304,8 @@ export type InitialRSCPayload = {
   f: FlightDataPath[]
   /** missingSlots */
   m: Set<string> | undefined
-  /** GlobalError */
-  G: [React.ComponentType<any>, React.ReactNode | undefined]
+  /** GlobalError: [Component, styles] */
+  G: [React.ComponentType<any>, React.ReactNode]
```

Comment:
Shouldn't this just be the `GlobalErrorState` helper type we created? Once we labelled the tuple members, we can skip the comment.

Reply:
It will regress the type routes test, revert and keep the origin for now in the PR. will follow up this in the future
```
pnpm test test/integration/app-types/app-types.test.js

app type checking > production mode > should generate route types correctly and report link error
```